tests/site_upload/test_powerset_merge.py

```diff
@@ -209,6 +248,12 @@ def test_powerset_merge_single_upload(
             if study in item["Key"] and status == 200:
                 agg_df = awswrangler.s3.read_parquet(f"s3://{TEST_BUCKET}/{item['Key']}")
                 assert (agg_df["site"].eq(site)).any()
+                print(len(agg_df))
+                print(agg_df.iloc[0].to_list())
+                print(agg_df.iloc[-1].to_list())
+                assert expected_rows == len(agg_df)
+                assert first_row == agg_df.iloc[0].to_list()
+                assert last_row == agg_df.iloc[-1].to_list()
```

Comment:
One thing that's helpful with asserts is to add data to print if it fails with a comma, so you don't need the print statements.

Like:
```
                assert expected_rows == len(agg_df), len(agg_df)
                assert first_row == agg_df.iloc[0].to_list(), agg_df.iloc[0].to_list()
                assert last_row == agg_df.iloc[-1].to_list(), agg_df.iloc[-1].to_list()
```

Reply:
oh i had no idea, that's incredibly useful (though i meant to strip those out)